.github/workflows/test.yml

```diff
@@ -1,12 +1,13 @@
-name: CI
+name: Test
 on: [push, pull_request]
 
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.8, 3.9, "3.10"]
+        django-version: [">=3.2,<4", ">=4,<5"]
```

Comment:
I would also test against 3.2.x, as it's the most recent LTS.

Reply:
You mean Django? 

It does - You can see in https://github.com/OpenDataServices/json-data-ferret/runs/6608907132?check_suite_focus=true - open the "run pip install Django" step and you can see it installs Django 3.2.13.

https://github.com/OpenDataServices/json-data-ferret/runs/6608907261?check_suite_focus=true installs 4.0.4, so we are covering both 3.2 & 4.

I was careful to get both - Golab is currently on 3.2 and I don't want to be forced to upgrade that right now but we will have to at some point and I wanted to make sure we were ready.

Reply:
Ah yes, this is fine. I was thinking there was a 3.3, but there isn't.